### PR TITLE
Welcome animation improvements

### PR DESCRIPTION
**Pull request by goldjay**
[FEV-892](https://auderenow.atlassian.net/browse/FEV-892)

* Animation lines up from AnimatedChrome -> Chrome!

Animation iPhone 8+

### CODE REVIEW

File: FluStudy_au/src/ui/components/Chrome.tsx [hunk 1]
```diff
@@ -42,52 +44,83 @@ export default class Chrome extends React.PureComponent<Props> {
...
       <View style={styles.container}>
-        <ImageBackground
-          source={SPLASH_IMAGE}
+        <View
           style={[
-            { alignSelf: "stretch" },
-            !!this.props.splashImage && {
-              aspectRatio: SPLASH_RATIO,
-              width: "100%",
+            {
```
Comment (jennya) [inline FluStudy_au/src/ui/components/Chrome.tsx]: For consistency & readability, could you move styles to the StyleSheet definition below where possible?

File: FluStudy_au/src/ui/components/AnimatedChrome.tsx [hunk 2]
```diff
@@ -36,7 +39,7 @@ interface Props {
   splashImage?: string;
 }
 
-const screenHeight = Dimensions.get("window").height;
+const screenWidth = Dimensions.get("window").width;
```
Comment (jennya) [inline FluStudy_au/src/ui/components/AnimatedChrome.tsx]: Seems a little weird to have this defined here and screenHeight defined in render below

Comment (goldjay) [reply FluStudy_au/src/ui/components/AnimatedChrome.tsx]: @jennya Gotcha, I'll just move it to render

File: FluStudy_au/src/ui/components/Chrome.tsx (hunk 1 above)
Comment (goldjay) [reply FluStudy_au/src/ui/components/Chrome.tsx]: @jennya Sure, no problem

**Pull request merged**